R/mod_data_management.R

```diff
@@ -76,7 +76,7 @@ mod_data_management_server <- function(id, db_conn, filter_data) {
       
       validate(
         need(
-          data_exists <- filter_data()$filter_data %>%
+          filter_data()$filter_data %>%
```

Comment:
Sorry, no, what I meant was

```
filter_data()$filter_data %>%
            dplyr::tally() %>%
            dplyr::pull(n) > 0
```

Can be replaced just with `filter_data()$filter_data`

Reply:
the expression has to return TRUE. using only the `filter_data()$filter_data` didn't work